lib/generators/provider/templates/spec/spec_helper.rb

```diff
@@ -2,12 +2,17 @@
   require 'simplecov'
   SimpleCov.start
 end
-
+<% if options[:vcr] %>
+VCR.configure do |config|
+  config.ignore_hosts 'codeclimate.com' if ENV['CI']
+  config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
+end
+<% else %>
 # Uncomment in case you use vcr cassettes
 # VCR.configure do |config|
 #   config.ignore_hosts 'codeclimate.com' if ENV['CI']
 #   config.cassette_library_dir = File.join(ManageIQ::Providers::<%= class_name %>::Engine.root, 'spec/vcr_cassettes')
 # end
-
+<% end %>
```

Comment:
sometimes fresh 🍌 is better than DRY 🍌 

I'd keep my simpler if/else/end version.